### PR TITLE
[zh]Sync content/zh/docs/releases/supported-releases/index.md

### PR DESCRIPTION
**Pull request by yanrongshi**
Please provide a description for what this PR is for.

And to help us figure out who should review this PR, please 
put an X in all the areas that this PR affects.

- [ ] Configuration Infrastructure
- [x] Docs
- [ ] Installation
- [ ] Networking
- [ ] Performance and Scalability
- [ ] Policies and Telemetry
- [ ] Security
- [ ] Test and Release
- [ ] User Experience
- [ ] Developer Infrastructure

### CODE REVIEW

File: content/zh/docs/releases/supported-releases/index.md [hunk 1]
```diff
@@ -11,10 +11,10 @@ test: n/a
 
 此页面列出了当前支持的版本的状态，时间表和策略。受支持的 Istio 版本包括处于维护窗口期以及为安全问题和错误提供了补丁的版本。Minor 版本中的补丁版本不包含向后的兼容性。
 
-* [支持策略](#support-policy)
-* [命名方案](#naming-scheme)
-* [Istio 版本的支持状态](#support-status-of-istio-releases)
-* [没有已知常见漏洞（CVEs）的版本](#releases-without-known-Common-Vulnerabilities-and-Exposures)
+- [支持策略](#support-policy)
+- [命名方案](#naming-scheme)
+- [Istio 版本的支持状态](#support-status-of-istio-releases)
```
Comment (zirain) [inline content/zh/docs/releases/supported-releases/index.md:16]: still not working

Comment (yanrongshi) [reply content/zh/docs/releases/supported-releases/index.md:16]: > still not working

What should the correct one be? bosses

Comment (zirain) [reply content/zh/docs/releases/supported-releases/index.md:16]: same as L44, should be `support-status-of-Istio-releases`

File: content/zh/docs/releases/supported-releases/index.md [hunk 2]
```diff
@@ -11,10 +11,10 @@ test: n/a
...
+- [没有已知常见漏洞（CVEs）的受支持版本](#supported-releases-without-known-common-vulnerabilities-and-exposures-cves)
```
Comment (zirain) [inline content/zh/docs/releases/supported-releases/index.md:17]: same as L64, should be `releases-without-known-Common-Vulnerabilities-and-Exposures`

Comment (yanrongshi) [reply content/zh/docs/releases/supported-releases/index.md:17]: image

Comment (zirain) [reply content/zh/docs/releases/supported-releases/index.md:17]: 要么你两处一起改，否则跳转会失效

Comment (yanrongshi) [reply content/zh/docs/releases/supported-releases/index.md:17]: > 要么你两处一起改，否则跳转会失效

您说的两处指的是？

Comment (yanrongshi) [reply content/zh/docs/releases/supported-releases/index.md:17]: 链接我也更新了

Comment (zirain) [reply content/zh/docs/releases/supported-releases/index.md:17]: L17 与 L64

Comment (yanrongshi) [reply content/zh/docs/releases/supported-releases/index.md:17]: > L17 与 L64

英文没有同步，我不确定是否需要改，我将重新提交一个 pr，已确定这个问题

Comment (yanrongshi) [reply content/zh/docs/releases/supported-releases/index.md:17]: 您说的是对的，我确认过，英文在两处都有更改

File: content/zh/docs/releases/supported-releases/index.md (hunk 1 above)
Comment (zirain) [reply content/zh/docs/releases/supported-releases/index.md:16]: 这里还有问题

Comment (windsonsea) [reply content/zh/docs/releases/supported-releases/index.md:16]: Looks the anchors are same as en source now:
```
- [Support policy](#support-policy)
- [Naming scheme](#naming-scheme)
- [Support status of Istio releases](#support-status-of-istio-releases)
- [Supported releases without known Common Vulnerabilities and Exposures (CVEs)](#supported-releases-without-known-common-vulnerabilities-and-exposures-cves)
```

**Review by windsonsea — APPROVED** [review]

**Pull request merged**